src/protos/io/defang/v1/fabric.proto

```diff
@@ -106,9 +106,10 @@ message TrackRequest {
 }
 
 enum Behavior {
-  DEVELOPMENT = 0;
-  STAGING = 1;
-  PRODUCTION = 2;
+  BEHAVIOR_UNSPECIFIED= 0;
```

Comment:
Just `UNSPECIFIED`, otherwise will get doubled. Also space before `=`

Reply:
Apparently we cannot use the same name across different ENUMs:

```
io/defang/v1/fabric.proto:390:3:symbol "io.defang.v1.UNSPECIFIED" already defined at io/defang/v1/fabric.proto:109:3; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
make: *** [Makefile:13: protos/io/defang/v1/fabric.pb.go] Error 100
```

Reply:
🤯 you're right https://protobuf.dev/programming-guides/dos-donts/#unspecified-enum

Reply:
I can swap the order: UNSPECIFIED_BEHAVIOR, so it becomes BEHAVIOR_UNSPECIFIED_BEHAVIOR above, slightly better to read